CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ UNRELEASED
 * [ [#1086](https://github.com/digitalfabrik/integreat-cms/issues/1086) ] Provide correct URL for POI
 * [ [#1247](https://github.com/digitalfabrik/integreat-cms/issues/1247) ] Update translation status on source status changes
 * [ [#1251](https://github.com/digitalfabrik/integreat-cms/issues/1251) ] Fix change detection in page form
+* [ [#1260](https://github.com/digitalfabrik/integreat-cms/pull/1260) ] Fix Firebase messaging
```

Comment:
```suggestion
* [ [#1260](https://github.com/digitalfabrik/integreat-cms/issues/1260) ] Fix Firebase messaging
```
Unfortunately, our regex only handles links to `issues`... (and github redirects to the correct PR)

Reply:
Okay :D